### DISCUSSION TITLE
Disable Select All - Find All Refs.

### DISCUSSION THREAD

**Discussion by MrgSub** [zed-industries/zed#18947] [Feature Requests]
Is there any way to disable select in https://zed.dev/docs/outline-panel#find-all-references panel? I just want to see the references not select the first one, I want the input to be in focus so I can search again

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
